src/Compilers/VisualBasic/Test/Emit/CodeGen/AnonymousTypesCodeGenTests.vb

```diff
@@ -156,14 +156,21 @@ End Module
     </file>
                          </compilation>
 
+            Dim innerCount = 10
+            Dim streamList = new List(Of MemoryStream)()
+            For i = 0 To innerCount - 1
+                streamList(i) = new MemoryStream()
+            Next
+
             For i = 0 To 100
```

Comment:
Had the same thought.  This test already takes 39 seconds to run on Jenkins.  Was curious how much the memory allocation factored into that.